app/models/user.rb

```diff
@@ -0,0 +1,27 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable, :confirmable,
+         :recoverable, :rememberable, :validatable
+  validates :phone_number, length: { is: 11 }
+  validate :password_lower_case
+  validate :password_uppercase
+  validate :password_special_char
+
+  def password_uppercase
+    return if !!password.match(/\p{Upper}/)
+    errors.add :password, 'must contain at least 1 uppercase '
+  end
+
+  def password_lower_case
+    return if !!password.match(/\p{Lower}/)
```

Comment:
add extra line after return, if we have two or more then we will add it after the last one e.g:
return if condition A
return if condition B

//Code